src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -0,0 +1,93 @@
+import PositionModel from '../base/PositionModel';
+import FixCollection from './Fix/FixCollection';
+import StandardRouteCollection from './StandardRoute/StandardRouteCollection';
+
+/**
+ * @class NavigationLibrary
+ */
+export default class NavigationLibrary {
+    /**
+     * @constructor
+     * @for NavigationLibrary
+     * @param airportJson {object}
+     */
+    constructor(airportJson) {
+        /**
+         *
+         *
+         * @property _referencePosition
+         * @type {PositionModel}
+         * @default null
+         */
+        this._referencePosition = null
+
+        /**
+         *
+         *
+         * @property _sidCollection
+         * @type {StandardRoute}
+         * @default null
+         */
+        this._sidCollection = null;
+
+        /**
+         *
+         *
+         * @property _starCollection
+         * @type {StandardRoute}
+         * @default null
+         */
+        this._starCollection = null;
+
+        this.init(airportJson);
+    }
+
+    /**
+     * Lifecycle method, should be run only once on instantiation
+     *
+     * Set initial class properties
+     *
+     * @for NavigationLibrary
+     * @method init
+     */
+    init(airportJson) {
+        const { fixes, sids, stars } = airportJson;
+
+        this._referencePosition = new PositionModel(airportJson.position, null, airportJson.magnetic_north);
+
+        FixCollection.addItems(fixes, this._referencePosition);
```

Comment:
Correct.  I _really_ wanted to make this like everything else as an instance class.  The main reason is because of the `StandardRoute` class structure.  The `StandardRouteWaypointModel` needs access to the `FixCollection` and, to be completely honest, I didn't want to have to pass a `fixCollection` instance all the way down the chain.  I tried, and it was a big pain in the ass and very confusing.  

So it remains static.  Still not happy about it.  But it is tested and all of its `init` and `reset` logic are contained (or will be) within the `navigationLibrary`.